src/doom/d_main.c

```diff
@@ -1240,6 +1240,42 @@ static void LoadIwadDeh(void)
             I_Error("Failed to load chex.deh needed for emulating chex.exe.");
         }
     }
+
+    if (gamemission == doom2 && W_CheckNumForName("M_RDTHIS") < 0 && W_CheckNumForName("M_EPISOD") < 0 && W_CheckNumForName("M_EPI1") < 0
```

Comment:
These are very long lines. Can you please keep to the 80 column limit as described in HACKING.md?

One suggestion to make this clearer: you could define a function called `IsFrenchIWAD` that just `return`s this expression. Then it's clear what the lump checks are for.